src/core/brand/SvgRoundel.tsx

```diff
@@ -0,0 +1,47 @@
+export type SvgRoundelProps = {
+	/**
+	 * The text colour.
+	 *
+	 * This should probably come from
+	 * [`@guardian/src-foundation`](https://theguardian.design/2a1e5182b/p/2668c8-code).
+	 */
+	textColor?: string;
+	/**
+	 * The roundel's background colour.
+	 *
+	 * This should probably come from
+	 * [`@guardian/src-foundation`](https://theguardian.design/2a1e5182b/p/2668c8-code).
+	 */
+	bgColor?: string;
```

Comment:
Can we go with `backgroundColor` for consistency with how the colour tokens are named?

Reply:
fixed in 8eebde9